scripts/install_emsdk.js

```diff
@@ -52,6 +52,8 @@ function check() {
     }
 }
 
-if (!check()) {
-    upgrade();
+if (!process.env.PSP_SKIP_EMSDK_INSTALL) {
```

Comment:
I understand you don't want to always auto-install emsdk as a side effect of other commands, but putting this check here disables installation globally including when it is set in `.perspectiverc`.   Can we do this where `install_emsdk` is invoked as a side effect instead (presumably `post_install`?)

I'd like to say no developer is stupid enough to get confused by this, but I just was, for an embarrassingly long time.

Reply:
If you set `PSP_SKIP_EMSDK_INSTALL`, a very specific env var, you will not install `emsdk`. Otherwise it will operate as before.

Reply:
I cited an explicit example of how this fails (env vars injected through `.perspectiverc`), and it is [so easy to fix](https://docs.npmjs.com/cli/v8/commands/npm-install#ignore-scripts)